### PR TITLE
1. Created the sidebar with logos in it. 2. Also created the links to…

### PR DESCRIPTION
**Pull request by Um-Brehon**
…wards other pages

### CODE REVIEW

**Review by KeerhemFR — DISMISSED** [review]
Everything seems okay

File: src/Components/Navbar/Index.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  return (
+    <>
+      <Link to="/" className="yellowBars" onClick={showSidebar}>
+        <img src={YellowBars} alt="" />
+      </Link>
+
+      <nav className={sidebar ? 'navMenuActive' : 'navBar'}>
+        <Link to="/" className="menuLogo" onClick={showSidebar}>
+          <img src={NavbarLogo} alt="" />
```
Comment (Hacene010) [inline src/Components/Navbar/Index.jsx]: Put something into the alt plz!!!

File: src/Components/Navbar/Index.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        </Link>
+        <div className="burgerImg">
+          <img src={WildHeroesBurger} alt="" />
```
Comment (Hacene010) [inline src/Components/Navbar/Index.jsx]: Same here !

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Still some evils to fix.

**Review by KeerhemFR — DISMISSED** [review]
Seems okay to me

File: src/App.jsx [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+import './reset.css';
+import './App.css';
+
 function App() {
-  return <h1>Hello there!</h1>;
+  return (
+    <>
+      <NavBar />
+      <Switch>
+        <Route path="/" exact component={Home} />
```
Comment (Hacene010) [inline src/App.jsx:19]: Does it work like that (with the "exact" after the path )? because I know that we write it like this ( " exact path=' /' )

**Review by Hacene010 — DISMISSED** [review]
Ok but check the comment plz

**Review by KeerhemFR — APPROVED** [review]
It's okay for me.

**Review by Hacene010 — APPROVED** [review]
ok

**Pull request merged**